### PR TITLE
Reverting to 3.12 until RTD supports 3.13

### PR DESCRIPTION
**Pull request by tabedzki**

### CODE REVIEW

File: .readthedocs.yml
Comment (funkyfuture) [inline .readthedocs.yml:1]: amending a list item `furo` to `python.install` should solve the build failure.

**Review by funkyfuture — CHANGES_REQUESTED** [review]
please don't change builds (Dockerfiles' base images) and the workflow. they have no impact on the RTD build.

**Comment by tabedzki** [comment]
If the pyproject.toml and the poetry.lock are set for Python 3.13, would the build for RTD fail due to being 3.12?

**Comment by funkyfuture** [comment]
> If the pyproject.toml and the poetry.lock are set for Python 3.13, would the build for RTD fail due to being 3.12?

yes, because Python package installers would deny an installation of the project itself.

i'm closing this pr in favor of #154.

**Pull request closed without merging**